pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -13,6 +13,12 @@ const (
 	AccessModePrivate string = "private"
 )
 
+// Field to indicate whether support for load balancing is needed
```

Comment:
Nit: add a "." to the end of the sentence.

Reply:
done, thanks.